### PR TITLE
Add sqlite source

### PR DESCRIPTION
**Pull request by jhoekx**

### CODE REVIEW

File: docs/source/sqlite.asciidoc [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+There is a possibility to add a quality column.
+
+In this case the data query changes:
+
+```toml
+data_query = "select timestamp, value, quality from Data where name = ? and ts >= ? and ts < ?"
+```
+
+Where `quality` represents the column that contains the data point quality of the ADODB source.
```
Comment (WouterWouters) [inline docs/source/sqlite.asciidoc:270]:
```suggestion
Where `quality` represents the column that contains the data point quality of the SQLite Data table.
```

**Review by WouterWouters — APPROVED** [review]

File: docs/source/sqlite.asciidoc (hunk 1 above)
Comment (jhoekx) [reply docs/source/sqlite.asciidoc]: Fixed.

**Comment by WouterWouters** [comment]
Added a sqlite source and tested the different queries. 
list query: OK
metadata query: OK
dictionary: OK
Data query: I created a DB with limited data points with a timestamp as the ISO standard provided. 
```
1|9a68f02b-abaf-4a31-9f9d-46a2c9638c95|2021-01-01T00:00:00+00:00|1.0
2|9a68f02b-abaf-4a31-9f9d-46a2c9638c95|2021-01-01T01:00:00+00:00|2.0
3|9a68f02b-abaf-4a31-9f9d-46a2c9638c95|2021-01-01T02:00:00+00:00|5.0
7|9a68f02b-abaf-4a31-9f9d-46a2c9638c95|2021-01-02T00:00:00+00:00|5.0
8|9a68f02b-abaf-4a31-9f9d-46a2c9638c95|2021-01-03T00:00:00+00:00|7.0
```

But the Chart in Timeseer would not load, an evaluation of the source would result in an event frame with 'no data'.

My config
```
[source.sqlite]
type = "sqlite"
connection_string = "storage/database.sqlite"
query_timeout_seconds = 0
list_query = "select series_id, description, units, interpolationType, dataType, dictionaryName from RealMetadata"
list_columns = ["series name", "description", "unit", "interpolation type", "data type", "dictionary name"]
metadata_query = "select description, units, interpolationType, dataType, dictionaryName from Metadata where series_id = ?"
metadata_columns = ["description", "unit", "interpolation type", "data type", "dictionary name"]
dictionary_query = "select value, label from Dictionary where series_id = ?"
data_query = "select ts, value from Data where series_id = ? and ts >= ? and ts < ?"
enable_trace_logging = false
```

Not sure if I did something wrong or there is a bug in the code.

**Comment by jhoekx** [comment]
Inside Timeseer we use a different datetime conversion (that optimizes for storage) than the one used by Kukur, so this source cannot be used for data there.

**Pull request merged**